config.toml

```diff
@@ -12,8 +12,8 @@ theme = "hugo-book"
   url = "https://uapi-group.org/specifications"
   weight = 11
 [[menu.after]]
-  name = "Kernel Features"
-  url = "https://uapi-group.org/kernel"
+  name = "Kernel Features Ideas"
```

Comment:
I think it should be "Kernel Feature Ideas" so singular, "Feature" instead of plural "Features"?